datastore/common/schemas/031aws-tables-create.sql

```diff
@@ -707,6 +707,167 @@ CREATE TABLE raw_aws_network_acl_subnets (
         ON UPDATE CASCADE
 );
 
+-- ----------------------------------------------------------------------
+-- AWS IAM
+-- ----------------------------------------------------------------------
+
+CREATE TABLE raw_aws_iam_document (
+    tool_run_id UUID NOT NULL,
+    attachment_id TEXT NOT NULL,
+    secondary_id TEXT NOT NULL,
+    doc_version TEXT NOT NULL,
+    PRIMARY KEY (tool_run_id, attachment_id, secondary_id, doc_version),
+    FOREIGN KEY (tool_run_id)
+        REFERENCES tool_runs(id)
+        ON DELETE CASCADE
+        ON UPDATE CASCADE
+);
+
+
+CREATE TABLE raw_aws_iam_group (
+    tool_run_id UUID NOT NULL,
+    id TEXT NOT NULL,
+    arn TEXT NOT NULL,
+    name TEXT NOT NULL,
+    create_date TEXT NOT NULL,
+    path TEXT NOT NULL,
+    tags JSON NOT NULL,
```

Comment:
If you have a JSON column that is part of a primary key, I'm pretty sure it has to be `JSONB` so it can actually index them.